src/UNL/UCBCN/Event/RecurringDates.php

```diff
@@ -40,9 +40,15 @@ public function getSQL()
                         'ORDER BY recurrence_id ASC;';
                 }
             } else {
-                return 'SELECT id FROM recurringdate WHERE ' .
-                    'event_datetime_id = ' . (int)($this->options['event_datetime_id']) . ' AND unlinked = 0 AND ongoing = 0 ' .
-                    'ORDER BY recurrence_id ASC;';
+                if (array_key_exists('with_ongoing', $this->options)) {
+                    return 'SELECT id FROM recurringdate WHERE ' .
+                        'event_datetime_id = ' . (int)($this->options['event_datetime_id']) . ' AND unlinked = 0 ' .
+                        'ORDER BY recurrence_id ASC;';
```

Comment:
The `ORDER BY` clause is repeated for both cases, it might be worth moving this outside of the IF statement.